_tutorials/2016-05-22-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,101 @@
+---
+published: false
+date: "2016-05-22 13:49 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64-quickstart"
+author: Lisa Roach
+excerpt: "Getting started with Cisco's XRv64 Vagrant Instance."
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - XRv64
+---
+### Quick Start Using XRv64 Box image
+Quick guide to get up and running with XRv64 and vagrant/Virtualbox.
+ 
+#### Requirements:
+- [Vagrant](https://www.vagrantup.com/) and [Virtualbox](https://www.virtualbox.org/) installed on your laptop
+
+#### Single Node:
+
+	vagrant init xrv64
+  ![vagrant init xrv64](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_vagrant_init.png)
+   
+	vagrant box add --name xrv64 <REPLACE THIS WITH PUBLIC VAGRANT BOX URL> --force
+    
+  ![vagrant box add](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_vagrant_add.png)
+  
+	vagrant up 
+
+This will take some time, possibly over 10 minutes once you see the "Waiting for machine to boot" message.  Look for the green "vagrant up" welcome message to confirm the machine has booted:
+	
+   ![vagrant up](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_vagrant_up_s.png)
+    
+
+
+Now you have two options for accessing the Vagrant instance:
+
+1. Access the XR Linux shell:
+  		
+       vagrant ssh
+
+  ![vagrant ssh](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_vagrant_ssh_s.png)
+
+2.  Access XR Console:
+ 
+        ssh -p <forwarded port> vagrant@127.0.0.1
+
+   * Replace `<forwarded port>` with the port forwarded for your 22 port
+
+      * You can check which ports were forwarded with the command:
+
+		    vagrant port
+      
+    * The password is “vagrant” 
+    
+  ![ssh console](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_ssh_console_s.png)
+
+#### Multi Node:
+
+![Topology](https://xrdocs.github.io/xrdocs-images/assets/tutorial-images/xrv64_topo_m.png)
+
+Copy the multi-node Vagrantfile: `cp xr-cp-vbox/Vagrantfile`
```

Comment:
Same question as earlier. Where is this Vagrantfile? How do you get it? What is xr-cp-vbox?

Add steps to make a user do a git clone of Rich's xr-cp-vbox repository (which will be open sourced) and then give a snippet of the Vagrantfile to show what it is doing exactly.